pkg/hubble/dropeventemitter/fake_recorder.go

```diff
@@ -0,0 +1,79 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
```

Comment:
The original copyright must be retained to comply with the license.
```suggestion
// SPDX-License-Identifier: Apache-2.0
// Copyright 2015 The Kubernetes Authors.
```

I personally wouldn't add the Cilium copyright for a handful of lines of code updated, and I _think_ it's fine to trim the license text. cc @xmulligan to confirm on thoses points.

Reply:
Yeah, I was confused on what the correct process here was. I originally retained the exact wording from client-go, ended up switching it to cilium one after linter complained. Updated this to kubernetes authors now.

Reply:
Now the linter is unhappy again :sweat_smile:

Reply:
Should we then add this file to the [goheader exception list](https://github.com/cilium/cilium/blob/e49c7b3ddaf7e53a91a722ce7a1a328230749afc/.golangci.yaml#L108-L111)?

Reply:
So add another entry like so ?
```
    - path: "pkg/hubble/dropeventemitter/fake_recorder.go"
      linters:
        - goheader
```

Reply:
Done